content/en/database_monitoring/setup_mongodb/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Setting up MongoDB
+kind: documentation
+description: Setting up Database Monitoring on a MongoDB cluster
+disable_sidebar: true
+---
+
+{{< beta-callout url="#" btn_hidden="true" >}}
+Database Monitoring for MongoDB is currently in private beta. If you are interested in participating, please reach out to your Datadog Customer Success Manager.
+{{< /beta-callout >}}
```

Comment:
I think there's a specific blue banner we want to use here. LMK if you don't know what I'm talking about